channeldb/payments_kv_store.go

```diff
@@ -119,17 +119,66 @@ var (
 
 // KVPaymentsDB implements persistence for payments and payment attempts.
 type KVPaymentsDB struct {
-	paymentSeqMx     sync.Mutex
-	currPaymentSeq   uint64
-	storedPaymentSeq uint64
-	db               *DB
+	// db is the underlying database implementation.
+	db kvdb.Backend
+
+	kvStoreConfig *paymentsdb.StoreOptions
+
+	// Sequence management for the kv store.
+	seqMu     sync.Mutex
+	currSeq   uint64
+	storedSeq uint64
```

Comment:
This commit is too big, we can break it into four smaller ones,
- introduce the above opt/cfg
- rename `paymentSeqMx`
- rename `currPaymentSeq`
- rename `storedPaymentSeq`

Reply:
done